### PR TITLE
maf_stats: remove unused parameter

### PR DESCRIPTION
**Pull request by bernt-matthias**
No idea what the tool does (the generated command line and the python script are super confusing) .. but species is not used.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/maf_stats/maf_stats.xml [hunk 1]
```diff
@@ -87,40 +76,40 @@
...
+    ========  ===========  ========
+    #species  nucleotides  coverage
+    ========  ===========  ========
+    hg18         30639      0.2372
+    rheMac2      7524       0.0582
+    panTro2      30390      0.2353
+    ========  ===========  ========
 
-        where **coverage** is the number of nucleotides divided by the total length of the provided intervals.
+where **coverage** is the number of nucleotides divided by the total length of the provider intervals.
```
Comment (bernt-matthias) [inline tools/maf_stats/maf_stats.xml:112]: I think provided was correct. Maybe more explicit like so:

```suggestion
where **coverage** is the number of nucleotides divided by the total length of the intervals provided in the BED file.
```

**Comment by bgruening** [comment]
@nekrut @martenson do you remember what this tool was supposed to do? And why the species filter is not used?

I guess it makes sense to filter species from a huge MAF file.

**Comment by bernt-matthias** [comment]
Guess this was intended to work similar to https://github.com/galaxyproject/tools-iuc/blob/main/tools/interval2maf/interval2maf.py .. 

Not sure how to proceed here. I have no time to implement it (an given the IMO horrible code I also would rather not touch it), but leaving an unused parameter seems also like no option.

**Comment by bgruening** [comment]
Agree. Happy to merge when @nekrut or @martenson don't object or don't respond:-)

**Review by bgruening — APPROVED** [review]

**Pull request merged**